ui/src/app/constants/api.constants.ts

```diff
@@ -0,0 +1,21 @@
+
```

Comment:
I would have named this file just `api.ts` because it's already in a folder named `constants`, but this is just a suggestion

Reply:
"Angular way"